posts/inside-rust/2021-03-17-compiler-team-march-steering-cycle.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+title: Rust Compiler March Steering Cycle
+author: Felix Klock
+description: "The compiler team's March steering cycle"
+team: The Compiler Team <https://www.rust-lang.org/governance/teams/compiler>
+---
+On [Friday, March 12th][mar-12-zulip-archive], the Rust Compiler team had a planning meeting for the March steering cycle.
+
+[mar-12-zulip-archive]: https://zulip-archive.rust-lang.org/238009tcompilermeetings/07070planningmeeting20210312.html
+
+(The cycle here is that every fourth Friday, the Rust compiler team decides how
+it is going to use its scheduled steering and design meeting time over the next
+three Fridays.)
+
+It wasn't until the end of the meeting that we realized that April 2nd is a holiday for many folks.
+But luckily we decided to schedule just two meetings anyway.
```

Comment:
Maybe just move this last (so all the dates are chronological) and say 

```suggestion
Since April 2nd is a holiday for many folks, we will not be holding a meeting on that day. 
```

Reply:
(i'm going to simplify my own life and adopt your change without moving its position in the post.)